src/parse.py

```diff
@@ -45,27 +45,39 @@ def parse_usbank(record):
         "date": record["Date"],
         "description": record["Name"],
         "amount": record["Amount"],
-        # no category :(
+        "category": None,  # No category
         "source": "usbank",
     }
 
 
+PARSERS = {
+    "amex": parse_amex,
+    "chase": parse_chase,
+    "capital_one": parse_capital_one,
+    "usbank": parse_usbank,
+}
+
+
 def default_parser(record):
     return record
 
 
-def get_parser(filename):
+def get_card_from_filename(filename):
```

Comment:
@velicanu how about this?

Reply:
I like it! Now we don't have to write another if statement when we add a card.

I think there's a syntax error tho, your `else` is at the wrong indent level.

Reply:
Or is that a pythonic `for-else` thing I'm unaware of? 🤔

Reply:
nvm... just googled it 😅 

TIL new python syntax 😄

Reply:
Yup!
https://docs.quantifiedcode.com/python-anti-patterns/correctness/not_using_else_in_a_loop.html